manifests/server/config.pp

```diff
@@ -35,8 +36,8 @@
 
       # Lets setup the base rules
       $local_auth_option = $version ? {
-        '8.1'   => 'sameuser',
-        default => undef,
+        '8.1'   => "sameuser ${local_auth_option_postgres}",
```

Comment:
If `local_auth_option_postgres` is set to `undef`, then `'8.1'` will be set to `"sameuser "` vs `"sameuser"`.